pkg/util/log/e2e/e2e.go

```diff
@@ -0,0 +1,30 @@
+package e2e
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"github.com/sirupsen/logrus"
+
+	"github.com/Azure/ARO-RP/pkg/metrics"
+	"github.com/Azure/ARO-RP/pkg/metrics/statsd/e2e"
+)
+
+type E2EHook struct {
+	logToMetrics e2e.E2ELogToMetrics
+}
```

Comment:
this doesn't need to be exported does it?

Reply:
No, it doesn't - no longer exporting struct.